package.json

```diff
@@ -18,12 +18,15 @@
     "serve": "node lib/index.js",
     "start": "nodemon",
     "test": "exit 0",
-    "prepublishOnly": "npm run build"
+    "format": "prettier --write .",
+    "format:check": "prettier --check .",
+    "prepublishOnly": "npm run build",
+    "postinstall": "mkdir -p elements; cd elements; git init; git remote add -f origin https://github.com/ory/elements.git; git pull --depth=1 origin main; git checkout v0.0.1-alpha.0; npm i; npm run initialize; npm run build:markup; cd ..; npm i ./elements/packages/markup"
```

Comment:
originally with this setup i could do some pre-setup like setting `git config core.sparsecheckout true`.
But i think cloning like this would also be fine.